### PR TITLE
This removes grammatical mistakes from the code.

### PR DESCRIPTION
**Pull request by skanda890**
- [X] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#coding) and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**
- [X] **Have you checked that there aren't other open [pull requests](https://github.com/marticliment/WingetUI/pulls) for the same changes?**
- [X] **Have you tested that the committed code can be executed without errors?**
- [X] **This PR is not composed of garbage changes used to farm GitHub activity to enter potential Crypto AirDrops.**
Any user suspected of farming GitHub activity with crypto purposes will get banned. Submitting broken code wastes the time of the contributors, who have to spend their free time reviewing, fixing, and testing code that does not even compile, breaks other features, or does not introduce any useful changes. Thank you for your understanding.

-----

This removes grammatical mistakes from the code.

### CODE REVIEW

**Review by skanda890 — COMMENTED** [review]
This must be fine.

File: .github/ISSUE_TEMPLATE/bug-issue.yml_new [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
   - type: checkboxes
     attributes:
-      label: Please confirm these before moving forward
+      label: Please confirm these before moving forward.
       description: Please confirm the following before posting your issue.
       options:
         - label: I have searched for my issue and have not found a work-in-progress/duplicate/resolved issue.
           required: true
-        - label: I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
+        - label: I have tested that this issue has not been fixed in the latest [(beta or stable) release].(https://github.com/marticliment/WingetUI/releases/)
```
Comment (marticliment) [inline .github/ISSUE_TEMPLATE/bug-issue.yml_new:14]: Dots cannot be added between the [] and () elements of a hyperlink

File: .github/ISSUE_TEMPLATE/bug-issue.yml_new [hunk 2]
```diff
@@ -1,57 +1,57 @@
...
+      placeholder: How can this issue be reproduced? List here, if known, the steps followed before this issue appeared.
     validations:
       required: false
   - type: textarea
     attributes:
-      label: UniGetUI Log
+      label: WingetUI Log
       render: "text"
-      placeholder: Paste here your UniGetUI logs. Click the button on the top-right corner of UniGetUI -> UniGetUI Log
+      placeholder: Paste here your WingetUI logs. Click the button on the top-right corner of WingetUI -> WingetUI Log.
```
Comment (marticliment) [inline .github/ISSUE_TEMPLATE/bug-issue.yml_new:36]: Please do not revert UniGetUI to WingetUI

File: .github/ISSUE_TEMPLATE/enhancement-improvement.yml [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
   - type: checkboxes
     attributes:
-      label: Please confirm these before moving forward
+      label: Please confirm these before moving forward.
       description: Please confirm the following before posting your issue.
       options:
         - label: I have searched for my feature proposal and have not found a work-in-progress/duplicate/resolved/discarded issue.
           required: true
-        - label: This improvement refers to an existing feature. If you want to suggest a new feature, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=new-feature&projects=&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%28Enter+your+description+here%29)
+        - label: This improvement refers to an existing feature. If you want to suggest a new feature, please use [this template].(https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=new-feature&projects=&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%28Enter+your+description+here%29)
```
Comment (marticliment) [inline .github/ISSUE_TEMPLATE/enhancement-improvement.yml]: Dots cannot be added between the [] and () elements of a hyperlink

File: .github/ISSUE_TEMPLATE/enhancement-improvement.yml_new [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
   - type: checkboxes
     attributes:
-      label: Please confirm these before moving forward
+      label: Please confirm these before moving forward.
       description: Please confirm the following before posting your issue.
       options:
         - label: I have searched for my feature proposal and have not found a work-in-progress/duplicate/resolved/discarded issue.
           required: true
-        - label: This improvement refers to an existing feature. If you want to suggest a new feature, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=new-feature&projects=&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%28Enter+your+description+here%29)
+        - label: This improvement refers to an existing feature. If you want to suggest a new feature, please use [this template].(https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=new-feature&projects=&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%28Enter+your+description+here%29)
```
Comment (marticliment) [inline .github/ISSUE_TEMPLATE/enhancement-improvement.yml_new]: Dots cannot be added between the [] and () elements of a hyperlink

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
   - type: checkboxes
     attributes:
-      label: Please confirm these before moving forward
+      label: Please confirm these before moving forward.
       description: Please confirm the following before posting your issue.
       options:
         - label: I have searched for my feature proposal and have not found a work-in-progress/duplicate/resolved/discarded issue.
           required: true
-        - label: This proposal is a completely new feature. If you want to suggest an improvement or an enhancement, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=enhancement&projects=&template=enhancement-improvement.yml&title=%5BENHANCEMENT%5D+%28Enter+your+description+here%29)
+        - label: This proposal is a completely new feature. If you want to suggest an improvement or an enhancement, please use [this template].(https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=enhancement&projects=&template=enhancement-improvement.yml&title=%5BENHANCEMENT%5D+%28Enter+your+description+here%29)
```
Comment (marticliment) [inline .github/ISSUE_TEMPLATE/feature-request.yml]: Dots cannot be added between the [] and () elements of a hyperlink

File: .github/ISSUE_TEMPLATE/feature-request.yml_new [hunk 6]
```diff
@@ -1,27 +1,27 @@
...
   - type: checkboxes
     attributes:
-      label: Please confirm these before moving forward
+      label: Please confirm these before moving forward.
       description: Please confirm the following before posting your issue.
       options:
         - label: I have searched for my feature proposal and have not found a work-in-progress/duplicate/resolved/discarded issue.
           required: true
-        - label: This proposal is a completely new feature. If you want to suggest an improvement or an enhancement, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=enhancement&projects=&template=enhancement-improvement.yml&title=%5BENHANCEMENT%5D+%28Enter+your+description+here%29)
+        - label: This proposal is a completely new feature. If you want to suggest an improvement or an enhancement, please use [this template].(https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=enhancement&projects=&template=enhancement-improvement.yml&title=%5BENHANCEMENT%5D+%28Enter+your+description+here%29)
```
Comment (marticliment) [inline .github/ISSUE_TEMPLATE/feature-request.yml_new]: Dots cannot be added between the [] and () elements of a hyperlink

File: .github/ISSUE_TEMPLATE/hard-crash.yml_new [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
       description: Please confirm the following before posting your issue.
       options:
-        - label: I have tried deleting a folder named `.wingetui` placed on the user directory
+        - label: I have tried deleting a folder named `.wingetui` placed on the user directory.
           required: true
-        - label: I have tried reinstalling UniGetUI
+        - label: I have tried reinstalling UniGetUI.
           required: true
-        - label: I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
+        - label: I have tested that this issue has not been fixed in the latest [(beta or stable) release].(https://github.com/marticliment/WingetUI/releases/).
```
Comment (marticliment) [inline .github/ISSUE_TEMPLATE/hard-crash.yml_new]: Dots cannot be added between the [] and () elements of a hyperlink

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -1,10 +1,10 @@
 <!-- Provide a general summary of your changes in the title above -->
 
-- [ ] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#coding) and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**
+- [ ] **I have read the [contributing guidelines].(https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#coding) and I agree with the [Code of Conduct].(https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**
```
Comment (marticliment) [inline .github/PULL_REQUEST_TEMPLATE.md]: Dots cannot be added between the [] and () elements of a hyperlink

File: .github/PULL_REQUEST_TEMPLATE.md_new [hunk 9]
```diff
@@ -1,10 +1,10 @@
 <!-- Provide a general summary of your changes in the title above -->
 
-- [ ] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#coding) and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**
+- [ ] **I have read the [contributing guidelines].(https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#coding) and I agree with the [Code of Conduct].(https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**
```
Comment (marticliment) [inline .github/PULL_REQUEST_TEMPLATE.md_new]: Dots cannot be added between the [] and () elements of a hyperlink

File: .github/ISSUE_TEMPLATE/bug-issue.yml_new (hunk 1 above)
Comment (marticliment) [inline .github/ISSUE_TEMPLATE/bug-issue.yml_new]: Dots cannot be added between the [] and () elements of a hyperlink

File: .github/ISSUE_TEMPLATE/bug-issue.yml_new (hunk 2 above)
Comment (marticliment) [inline .github/ISSUE_TEMPLATE/bug-issue.yml_new]: Please do not revert UniGetUI to WingetUI

**Review by marticliment — CHANGES_REQUESTED** [review]

**Comment by skanda890** [comment]
This also solves #2343

**Review by marticliment — APPROVED** [review]

**Pull request merged**